Week3/homework/3-step3.js

```diff
@@ -3,23 +3,53 @@
 // use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
   // add your code here
-  return str;
+  var strOut = "";
+  for (var i = 0; i < num; i++) {
+    strOut = strOut + str;
+  }
+  return strOut;
 }
 
 console.log('for', repeatStringNumTimesWithFor('abc', 3));
 
 // use a 'while' loop
 function repeatStringNumTimesWithWhile(str, num) {
   // add your code here
-  return str;
```

Comment:
The idea is to thing about it in the same way you did it for the _for_ loop.
First, you initialise the variable to be returned and the index used to loop.

_var strOut = "";
var i = 0;_

Then, while index is smaller then the maximum number given as param, namely _num_, you add the _str_ given as param to the current strOut.

 _while (i < num) {
    strOut = strOut + str;
    i++;
  }_

 and then return the strOut (which now will have the given string _str_ repeated _num_ times)
 
_return strOut;_